README.md

```diff
@@ -101,6 +101,29 @@ from the new application's `apps.py` file.
    Valid values for `verbosity` are `"queryPlanner"` (default),
    `"executionStats"`, and `"allPlansExecution"`.
 
+## Why are we building a MongoDB-Django integration?
```

Comment:
Putting this background information amid technical details is out of place. I wouldn't personally include this information in this document.

Reply:
What about moving it to the top of the document?